server/imports/fixtures/orders.js

```diff
@@ -130,17 +130,20 @@ export default function () {
     requiresShipping: true,
     shipping: [{
       shopId: getShopId(),
+      address: getAddress({ isShippingDefault: true }),
```

Comment:
These missing props were causing test failures. I'm pretty sure they were always improperly missing, but for some reason the old SS package didn't catch the errors I guess.